### ISSUE TITLE
$ZITADEL_DOMAIN variable prevents easy copy paste of API Doc curls

### ISSUE THREAD

**Issue by mffap** [zitadel/zitadel#5374]
As a developer, I want be able to copy/paste curl commands from the API reference so that I can test out the API immediately.

```[tasklist]
### Acceptance Criteria
- [ ] ...
```

**Comment by mffap** [comment]
@hifabienne I've opened this issue without a solution proposal for tracking and discussion. From a usability perspective this should be resolved somehow, ideally by letting readers populate the variable or by using environment variables. Both cases do not seem to work out of the box.

**Comment by hifabienne** [comment]
Yes, I tried to include the var at the moment, but because the request does not show with double quotes, but we couldn't figure out right now how to change that.
A solution would be if @peintnermax can change the domain field from dropdown to input field so the user can enter his own domain, and we take it for all the requests.

**Comment by mffap** [comment]
I guess this is just not possible with docusaurus, since the pages are statically generated. Closing this issue for now.
